srcpkgs/0ad-data/template

```diff
@@ -0,0 +1,20 @@
+# Template file for '0ad-data'
+pkgname=0ad-data
+version=0.0.23
+revision=1
+wrksrc="0ad-${version}-alpha"
+short_desc="Historically-based real-time strategy game (data files)"
+maintainer="Helmut Pozimski <helmut@pozimski.eu>"
+license="CC-BY-SA-3.0"
+homepage="https://play0ad.com"
+noarch=yes
```

Comment:
unde rrevision=

Reply:
I'm not sure what you mean, could you be more specific?